Task 5.3/src/routes/router.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from "express";
+import {getUser} from "../controller/getUser.js";
+import {createUser} from "../controller/createUser.js";
+import {removeUser} from "../controller/removeUser.js";
+import {getAutoSuggestUsers} from "../controller/suggestions.js";
+import {updateUser} from "../controller/updateUser.js";
```

Comment:
instead of having so many imports. you can make a index.js file in controller folder and then expose all controllers from that index.js file. We can discuss this over call if you didnt understand